app/templates/companies/new.hbs

```diff
@@ -30,17 +30,14 @@
 
   <div class="row">
     <div class="{{if isMobileApp 'small-7 large-8' 'small-12'}} columns">
-      {{t 'donor_details.company.crm_id'}}
+      {{t 'donor_details.company.crm_id'}} {{t 'donor_details.company.optional'}}
     </div>
   </div>
 
   <div class="row">
     <div class="{{if isMobileApp 'small-7 large-8' 'small-12'}} columns">
       {{#form-control}}
-        {{numeric-input value=crmId id="crm-id" name="crm-id" maxlength="21" required=true class='company-input' pattern="\d{1,8}"}}
-        <div class='input-error'>
```

Comment:
Any specific reason for removing validation message?

Reply:
Matt asked to remove it.

Reply:
@Darshan-Chauhan

Reply:
Ok, can you also remove the translation for validation message?

Reply:
and its a numeric field, so user wont be able to enter string characters. So removed it.

Reply:
> Ok, can you also remove the translation for validation message?

Sure, Actually i used the same translation for placeholder, so i'll change the key name.
Thanks

Reply:
Done 👍